plugins/alerts/api/alertModules/metric.js

```diff
@@ -273,9 +274,9 @@ function getAverageSessionDuration(app_id, period) {
 * @param {string} app_id - id of app
 * @return {object} promise
 */
-function getViewData(app_id) {
+function getViewData(app_id, viewId) {
     return new bluebird(function(resolve) {
-        fetch.getTimeObjForEvents("app_viewdata" + app_id, {app_id, qstring: {}}, {unique: "u", levels: {daily: ["u", "t", "s", "b", "e", "d", "n"], monthly: ["u", "t", "s", "b", "e", "d", "n"]}}, function(data) {
+        fetch.getTimeObj("app_viewdata" + crypto.createHash('sha1').update(app_id).digest('hex'), {app_id, qstring: {}}, {id: viewId, unique: "u", levels: {daily: ["u", "t", "s", "b", "e", "d", "n"], monthly: ["u", "t", "s", "b", "e", "d", "n"]}}, function(data) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4574400)